service/package.json

```diff
@@ -3,7 +3,10 @@
   "description": "Ozone service entrypoint",
   "main": "index.js",
   "dependencies": {
-    "@atproto/ozone": "0.1.1",
+    "@atproto/ozone": "0.1.11",
     "next": "14.0.1"
+  },
+  "volta": {
+    "node": "20.12.2"
   }
```

Comment:
Cool if we nix the volta reference here?  This is primarily for use in the docker build process, hoping to keep it pretty minimal.

Reply:
yh totally, I use it locally and my global node is still at v18 so it was giving me trouble when installing packages using v18.